2200094/14649신용카드.py

```diff
@@ -0,0 +1,16 @@
+T = int(input())
+
+for t in range(1, T+1):
+    number = list(map(int, input().split()))
+    odd = 0
+    even = 0
+    for i in range(0, 16, 2):
+        odd += number[i] * 2
+    for x in range(1, 15, 2):
+        even += number[x]
+        result = odd + even
```

Comment:
```suggestion
    result = odd + even
```
`result`는 `for`문 밖에서 한 번만 계산해주시면 됩니다!